drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -324,6 +324,13 @@ void at86rf2xx_set_cca_threshold(at86rf2xx_t *dev, int8_t value)
     at86rf2xx_reg_write(dev, AT86RF2XX_REG__CCA_THRES, value);
 }
 
+#ifdef MODULE_OPENTHREAD
+bool at86rf2xx_receiver_listening(at86rf2xx_t *dev)
+{
+    return (at86rf2xx_reg_read(dev, AT86RF2XX_REG__RX_SYN) & 0x80) ? false : true;
+}
+#endif
```

Comment:
Yes, but I see it a little bit difficult for this release :(. If I manage to link OT libs generated by their building system, I can disable all heavy features and continue with the development.
I will do my best.

Anyway, can we try to merge this for this release and then PR again the second branch?
Cheers!